### PR TITLE
solution

### PR DESCRIPTION
**Pull request by KrisMur**
[DEMO LINK](https://KrisMur.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

**Review by Pavlo000 — APPROVED** [review]
Great job!

File: src/api/goods.ts [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 };
 
-export const getRedGoods = () => {
-  return getAll()
-    .then(goods => goods); // get only red
+export const getRedGoods = async () => {
+  const goods = await getAll();
+  const goodsOne = goods;
+
+  return goodsOne.filter(good => good.color === 'red');
```
Comment (Olexandr6) [inline src/api/goods.ts:23]:
```suggestion

  return goods.filter(good => good.color === 'red');
```

File: src/api/goods.ts [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+
+  return response.json();
 }
 
-export const get5First = () => {
-  return getAll()
-    .then(goods => goods); // sort and get the first 5
+export const get5First = async () => {
+  const goods = await getAll();
+  const goodsOne = goods;
```
Comment (Olexandr6) [inline src/api/goods.ts:13]: Why do you need this additional variable?

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Good job🔥 Let's improve some small things👇

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**